Dockerfile

```diff
@@ -1,16 +1,23 @@
+#FROM ubuntu:14.04
+#RUN  apt-get update \
+#  && apt-get install -y wget \
+#  && rm -rf /var/lib/apt/lists/*
 FROM public.ecr.aws/amazonlinux/amazonlinux:2022
 
 RUN yum -y update
+RUN yum install -y wget
 RUN yum install -y tar net-tools curl vim unzip less libevent-devel openssl-devel python-devel libtool git patch make gcc --allowerasing
-RUN git clone https://github.com/pgbouncer/pgbouncer.git --branch "pgbouncer_1_15_0" && \
```

Comment:
Can we pls add the --branch value as parameter? Something like https://github.com/aws-samples/containerized-game-servers/blob/master/lyra/server/assets-image-multiarch/Dockerfile.template